chimera_app/data.py

```diff
@@ -54,7 +54,7 @@ def fetch_latest(self) -> None:
         """
         api_url = ('https://api.github.com/repos/chimeraos/'
                    'chimera-data/branches')
-        resp = requests.get(api_url)
+        resp = requests.get(api_url, timeout=1)
```

Comment:
1 second here seems quite low.

Reply:
I can make it larger, but is an API call and both the request and response should be fast, should be 10 seconds at maximum.